src/Common/src/CoreLib/System/IO/PathHelper.Windows.cs

```diff
@@ -33,9 +33,9 @@ internal static string Normalize(string path)
 
             // If we have the exact same string we were passed in, don't allocate another string.
             // TryExpandShortName does this input identity check.
-            string result = builder.AsSpan().Contains('~')
```

Comment:
Why not contains?

Reply:
Because it doesn't exist for char outside of CoreLib